src/components/atoms/GlobalStyles.js

```diff
@@ -323,14 +323,12 @@ const GlobalStyle = createGlobalStyle`
   input[type="button"],
   input[type="reset"] {
     display: inline-block;
-    cursor: pointer;
     padding: 0;
-
     font-family: inherit;
     text-decoration: none;
     white-space: nowrap;
     border: none;
-
+    cursor: url(${cursor}), auto !important;
```

Comment:
Can you offset the position so it is centered.

https://stackoverflow.com/questions/19560878/css-change-custom-cursor-image-origin-hotspot-to-center

https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

Reply:
Added to 4, 12